dist/LangtailNode.d.mts

```diff
@@ -0,0 +1,82 @@
+import * as Core from 'openai/core';
```

Comment:
are the dist files commited to git on purpose?

Reply:
yes they are.
Will remove them-we don't need  them anymore in git. Will use beta release of SDK instead

Reply:
removed now